NOM_visualizations/python/nom_data.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    omicsprocess_dataobj=func.get_id_results(
</pre><p>Change the name of the python objects to remove "omicsprocessing" please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/80/discussion/#comment-1783145826'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>